UI/webpack.config.js

```diff
@@ -172,21 +172,21 @@ if (TARGET !== "readme") {
     // Copy non-packed resources needed by the app to the release directory
     const CopyWebpackPluginOptions = {
         patterns: [
-            { context: "../node_modules", from: "dijit/icons/**/*", to: "." },
-            { context: "../node_modules", from: "dijit/nls/**/*", to: "." },
-            { context: "../node_modules", from: "dojo/nls/**/*", to: "." },
-            { context: "../node_modules", from: "dojo/resources/**/*", to: "." }
+            { context: "node_modules", from: "dijit/icons/**/*", to: "." },
+            { context: "node_modules", from: "dijit/nls/**/*", to: "." },
+            { context: "node_modules", from: "dojo/nls/**/*", to: "." },
+            { context: "node_modules", from: "dojo/resources/**/*", to: "." }
         ],
         options: {
             concurrency: 100
         }
     };
 
     const DojoWebpackPluginOptions = {
-        loaderConfig: require("./UI/js-src/lsmb/webpack.loaderConfig.js"),
-        environment: { dojoRoot: "UI/js" }, // used at run time for non-packed resources (e.g. blank.gif)
+        loaderConfig: require("./js-src/lsmb/webpack.loaderConfig.js"),
+        environment: { dojoRoot: "js" }, // used at run time for non-packed resources (e.g. blank.gif)
         buildEnvironment: { dojoRoot: "node_modules" }, // used at build time
-        locales: getPOFilenames("locale/po", ".po"),
```

Comment:
why does moving everything to UI/ cause the extension to change?

Reply:
We can grab the locales through `locale/*.PO` or `UI/src/locales/*.json`. In order to become less tied to the server, I chose to enumerate JSON instead.